app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryAdapter.java

```diff
@@ -72,6 +96,11 @@ public View getChildView(int groupPosition, final int childPosition,
         return convertView;
     }
 
+    /**
+     * returns total children count.
```

Comment:
Not comments it's commits. there has been a spelling mistake from my side. 
Regarding the best practices in git, do a quick google search. It will help you with your commit messages and commits. Since you have already done this, squashing is one option. I can squash them when merging. But that won't help you learn. So i expect you to do this on your own. Typically only one commit would have been sufficient for the comments in this module. You can find the PRs sent by other students for the reference.